tools/build/pg_config_stub/pg_config_vcpkg_stub.py

```diff
@@ -0,0 +1,37 @@
+"""
+We use vcpkg in github actions to build psycopg-binary.
+
+This is a stub to work as `pg_config --libdir` or `pg_config --includedir` to make it work with vcpkg.
+
+You will need install vcpkg and set `VCPKG_ROOT `env and run `vcpkg install libpq[client]:x64-windows-release`
+before you use this script
+"""
+
+import os
+import pathlib
+import sys
+import platform
+
+
+def main():
+    # only x64-windows
+    if not (sys.platform == "win32" and platform.machine() == "AMD64"):
+        raise Exception("this script should only be used in x64-windows")
+
+    what = sys.argv[1]
+
+    # on github actions it's `VCPKG_INSTALLATION_ROOT`
+    if "VCPKG_ROOT" in os.environ or "VCPKG_INSTALLATION_ROOT" in os.environ:
+        vcpkg_root = pathlib.Path(
+            os.environ.get("VCPKG_ROOT") or os.environ["VCPKG_INSTALLATION_ROOT"]
+        )
+        vcpkg_platform_root = vcpkg_root.joinpath(
+            "installed/x64-windows-release"
+        ).resolve()
+        if vcpkg_platform_root.exists():
+            if what == "--libdir":
+                if vcpkg_platform_root.joinpath("lib/libpq.lib").exists():
+                    print(str(vcpkg_platform_root.joinpath("lib")))
+            if what == "--includedir":
+                if vcpkg_platform_root.joinpath("include/libpq").exists():
+                    print(str(vcpkg_platform_root.joinpath("include")))

```

Comment:
no `main()` call?

Also, what happens if any of the `if` condition fails (line 24, line 31, unexpected `what`)? The script will succeed but not print anything. Maybe we should throw an exception in these cases

Reply:
Ah I see, you have included a pyproject. ok.

Reply:
> Also, what happens if any of the `if` condition fails (line 24, line 31, unexpected `what`)? The script will succeed but not print anything. Maybe we should throw an exception in these cases

OK, I'll add some error message below